services/dkp-insights-management/1.2.0/defaults/cm.yaml

```diff
@@ -30,11 +30,11 @@ data:
       imagePullPolicy: IfNotPresent
       registry: docker.io
       repository: mesosphere/insights-management
-      tag: v1.1.1
+      tag: v1.2.0
     insightsCRIngress:
       globalRateLimitAverageQPS: 100
       globalRateLimitBurst: 100
-    kubectlImage: bitnami/kubectl:1.29.6
+    kubectlImage: bitnami/kubectl:1.29.2
```

Comment:
why 1.29.2?

Reply:
Because this `cm.yaml` an autogenerated thing and ALL the image bumps should land into the `dkp-insights` repository.